ASIM/dev/ASimTester/filteringTest/ASimFilteringTest.py

```diff
@@ -77,6 +77,12 @@ def create_parameters_string(parser_file):
     return ','.join(paramsList)
 
 
+# Creating a string of the values in the list with colons between them
```

Comment:
>colons [](http://example.com/codeflow?start=51&length=6)

worth adding an example, and is that colons? or commas?

Reply:
Indeed it is commas, my mistake.
And added example.